usr/share/rear/lib/serial-functions.sh

```diff
@@ -0,0 +1,42 @@
+# Get available serial devices
+function get_serial_devices {
+    if [ -z $SERIAL_CONSOLE_DEVICES ]; then
+        serial_devices=$(ls /dev/ttyS[0-9]* /dev/hvsi[0-9]* | sort)
+        echo "$serial_devices"
+    else
+        $SERIAL_CONSOLE_DEVICES
+    fi
+}
+
+function build_cmdline {
+    # Enable serial console, unless explicitly disabled
+    if [[ ! "$USE_SERIAL_CONSOLE" =~ ^[yY1] ]]; then
```

Comment:
Use the `is_true` function, cf.
https://github.com/rear/rear/wiki/Coding-Style#relax-and-recover-functions

The code doe not match its comment
"Enable serial console, unless explicitly disabled"
because the `if`condition skips if $USE_SERIAL_CONSOLE is not true
but the comment tells it should only skip if $USE_SERIAL_CONSOLE is false
but "not true" is not same as "false", cf.
https://github.com/rear/rear/wiki/Coding-Style#relax-and-recover-functions
and the initial comment in default.conf currently at
https://github.com/rear/rear/blob/master/usr/share/rear/conf/default.conf#L38

That whole `if` block can be simplified into:
```
# Nothing to do 
is_true "$USE_SERIAL_CONSOLE"  || return 0
```